### ISSUE TITLE
Move `reference/features/field_properties.md` into `learn/core_concepts/documents.md`

### ISSUE THREAD

**Issue by guimachiavelli** [meilisearch/documentation#1330]
This is part of our current site restructure; more specifically, our effort to remove the Feature reference and move all its pages/content to more adequate places across the docs site.

Field properties is an important page with fundamental content, but as it is I believe it lacks context. It is intimately connected to the concept of documents and its fields, which is why I propose we:

1. remove `reference/features/field_properties.md`
2. place the majority of its content inside `learn/core_concepts/documents.md#fields

The biggest drawback I can think of is bloating a page inside core concepts, which is a page I believe we'd ideally like to be as accessible and easy as possible.

Thoughts, @meilisearch/docs-team?

**Comment by dichotommy** [comment]
`reference/features/field_properties.md` is a guide for `searchableAttributes` and `displayedAttributes`, which are both part of the settings object. Personally, I don't consider it super related to the concept of documents—but I see how it could be if working from a different understanding of what the core concepts pages are for.

I see the core concepts pages as being, well, conceptual. Not a guide for everything related to documents/indexes/relevancy, but a conceptual overview of those subjects. In reality, however, they've gradually become dumping grounds for all kinds of information. `reference/core_concepts/documents.md` contains all of the important information on primary keys, `reference/core_concepts/relevancy.md` contains a lot of key information on relevancy that isn't repeated elsewhere. This isn't really consistent with the idea of "core concepts" IMO and starts to stray away from the stated intention of the articles, making it less intuitive to find information (e.g. why is the lengthy explanation of the primary key in the "documents" article and not the "indexes" one? There's no logic.)

But I digress. **I would vote for maintaining this as a separate guide rather than folding it into one of the core concepts pages.** I think we should aim to keep the core concepts pages true to their name, insofar as that is possible and useful.

I would however like to change the name, as "field properties" seems like a useless layer of abstraction placed on top of what this is actually a guide for, which is searchable and displayed fields.

WDYT?

**Comment by guimachiavelli** [comment]
I think that's a fair assessment. I was more focussed on the task of getting stuff out of the reference than on the integrity of core concepts, but I agree with both your view of an ideal core concepts page (sort of a high-level summary of a concept) and what it currently is (everything relating to a concept e.g. documents that we feel is important and don't really know where to put).

Your suggestion of reframing this guide from something relating to fields and towards regular settings like `filterableAttributes` is great too.

I'm less clear on where we should place it, though. I assume you'd be ok with sticking it under Advanced Topics for this restructure?

**Comment by guimachiavelli** [comment]
What do you think about creating a Configuration heading under which we place:
1. Instance "Configuration" guide #1325
2. "Field properties" (renamed to "Searchable and displayed attributes")
3. "Distinct attributes"
4. "Synonyms"

At least at first, this would be a non-clickable heading under "Advanced Topics". The idea is to have a section grouping the dedicate guides we currently have that deal with configuring MeiliSearch's instances and indexes.

**Comment by guimachiavelli** [comment]
We have a yes from @maryamsulemani97. @dichotommy, any thoughts?

**Comment by dichotommy** [comment]
Yes, I support this! I see some potential flaws; mainly, it could be confusing to present instance options in the same subsection with index-specific settings. Long term, I think I would probably prefer to have something like this

Advanced guides
- Instance options
- Index settings
  - 
  
I also still worry that the term "instance" could be unclear for people arriving on our docs for the first time, but I haven't been able to think of anything better. "Startup options"? "Launch options"? "Customize Meilisearch"? These are all bad 😞 

Anyway I support making changes like this even if we have doubts since it's part of an iterative process that may move us closer to our goal.

TL;DR: Go for it 👍🏻
